src/ORM/DataObject.php

```diff
@@ -3314,7 +3312,7 @@ public function relObject($fieldPath)
             } elseif ($component instanceof Relation || $component instanceof DataList) {
                 // $relation could either be a field (aggregate), or another relation
                 $singleton = DataObject::singleton($component->dataClass());
-                $component = $singleton->dbObject($relation) ?: $component->relation($relation);
+                $component = $singleton->dbObject($relation) ?? $component->relation($relation);
```

Comment:
Unrelated refactor but we don't care about "falsy" values - the only possible falsy value is `null`.